c7n/resources/emr.py

```diff
@@ -306,6 +306,50 @@ def get_related_ids(self, resources):
 filters.register('network-location', net_filters.NetworkLocation)
 
 
+@filters.register('security-configuration')
+class EMRSecurityConfigurationFilter(ValueFilter):
```

Comment:
you should be able to do this with a related resource filter type?

Reply:
Yes it can be done with related resources, but AWS EMR Security Configuration doesn't have unique `Id`. it is only security configuration name